roles/openshift_node/tasks/apply_machine_config.yml

```diff
@@ -39,12 +39,12 @@
     content: "{{ (oc_get.stdout | from_json).spec.config }}"
     dest: "{{ temp_dir.path }}/worker_ignition_config.json"
 
-- name: Get machine-config-daemon image
+- name: Get machine-config-operator image
   command: >
-    oc get daemonset machine-config-daemon
+    oc get daemonset machine-config-operator
```

Comment:
this looks wrong:
```
12:37:51 [github.com/openshift/installer] ‹master*›  oc get daemonset machine-config-daemon
NAME                    DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR            AGE
machine-config-daemon   6         6         6       6            6           kubernetes.io/os=linux   16h
14:41:52 [github.com/openshift/installer] ‹master*›  oc get daemonset machine-config-operator
Error from server (NotFound): daemonsets.extensions "machine-config-operator" not found
```

Reply:
Fixed

Reply:
Ah, right, the daemonset name didn't change